### DISCUSSION TITLE
[Question] ddcutil and laptop panels

### DISCUSSION THREAD

**Discussion by abceleung** [rockowitz/ddcutil#359] [Q&A] [answered]
I just got a Thinkpad T14s Gen 3. Since the screen is only 45% NTSC, I am going to buy a better panel after doing some researching on panelook.com.
It is suggested that I calibrate the screen (with Displaycal) after the panel swap. However (correct me if I am wrong) I think it is not possible to calibrate the RGB values of Thinkpad like you do with a proper monitor with buttons. 
This brings me to ddcutil. 
- Is it true that ALL eDP panels doesn't work with ddcutil? 
  - If no, is it possible to find out what panels support ddcutil?

I don't find any specification related to DDC/CI in panelook.com.

**Comment by rockowitz** [comment, marked as answer]
eDP panels have a basic  I<sup>2</sup>C implementation sufficient for supplying the EDID on slave address x50.  DDC/CI on slave address x37 is simply not supported. There's nothing about about DDC in panelook.com because it's not implemented in the panel, it's implemented in the monitor's firmware.  eDP panels are designed to be used without extensive firmware.

However, screen calibration using DisplayCal is a separate issue.  While it's desirable to get the physical white point right on the monitor by adjusting its RGB settings, it's not required.  Unless you're using a VERY high end monitor and the corresponding calibration tools, calibration is entirely done by adjusting the video driver.

**Reply by abceleung** [reply]
Thank you for your answer!
